src/components/AppBody.jsx

```diff
@@ -140,6 +140,7 @@ class AppBody extends React.Component {
       setTimeout(() => parseReady = true, 1000);
       new Promise((resolve) => resolve(parser(snippet)))
       .then((AST) => {
+        console.log(JSON.stringify(AST));
```

Comment:
I think I removed an identical line from a different PR. Make sure you guys are creating your branches off master.